src/pyFAI/third_party/_local/transformations.py

```diff
@@ -512,11 +512,11 @@ def projection_matrix(
 
     """
     M = numpy.identity(4)
-    point = numpy.array(point[:3], dtype=numpy.float64, copy=False)
+    point = numpy.asarray(point[:3], dtype=numpy.float64)
     normal = unit_vector(normal[:3])
     if perspective is not None:
         # perspective projection
-        perspective = numpy.array(
+        perspective = numpy.asarray(
             perspective[:3], dtype=numpy.float64, copy=False
```

Comment:
Missing removal of `copy=False`:

```suggestion
        perspective = numpy.asarray(
            perspective[:3], dtype=numpy.float64,
```